File: Clases 2018/Clases-C#/Simulacro_parcial_2_2018 (creacion de bug e historico en transaccion)/VB_TPS_BugTracker/TPS_BugTracker/GUILayer/frmPrincipal.vb
Partial Public Class frmPrincipal
    Inherits Form

    Private Shared userLogin As Usuario = Nothing

    Public Sub New()
        InitializeComponent()
        toolStripStatusLabel1.Text = ""
    End Sub

    Private Sub frmPrincipal_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
        Dim login As frmLogin = New frmLogin(Me)
        login.ShowDialog()
    End Sub

    Public Sub actualizarUsuarioLogueado(ByVal user As Usuario)
        toolStripStatusLabel1.Text = "Usuario: " & user.nombre & " | Perfil: " + user.perfil
        userLogin = user
        Mnu_principal.Visible = True
    End Sub

    Private Sub ConsultaDeBugsToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ConsultaDeBugsToolStripMenuItem.Click
        Dim Bugs As frmBugs = New frmBugs()
        Bugs.ShowDialog()
    End Sub

    Private Sub SalirToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles SalirToolStripMenuItem.Click
        End
    End Sub

    Private Sub UsuariosToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles UsuariosToolStripMenuItem.Click
        Dim usuarios As frmUsuarios = New frmUsuarios()
        usuarios.ShowDialog()
    End Sub

    Public Shared Function obtenerUsuarioLogin() As Usuario
        Return userLogin
    End Function

    Private Sub SoporteToolStripMenuItem_Click(ByVal sender As Object, ByVal e As EventArgs)
    End Sub

    
End Class

File: Clases 2018/Clases-C#/ClasePractica2_2018/C#_TPS_InicioSesion/TPS_InicioSesion/TPS_InicioSesion/frmPrincipal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace TPS_InicioSesion
{
    public partial class frmPrincipal : Form
    {
        public frmPrincipal()
        {
            InitializeComponent();
        }

        // Credenciales Hardcodeadas!!
        public string usuario = "test";
        public string password = "test";

        private void frmPrincipal_Load(object sender, EventArgs e)
        {
            frmLogin login = new frmLogin();
            login.ShowDialog();
        }

        public bool ValidarCredenciales(string pUsuario, string pPassword)
        {
            if (pUsuario == usuario & pPassword == password)
                return true;
            else
                return false;
        }

    }
}
